akka-actor-typed-tests/src/test/scala/docs/akka/typed/StyleGuideDocExamples.scala

```diff
@@ -120,8 +120,8 @@ object StyleGuideDocExamples {
               context.log.debug(
                 "[{}] Starting repeated increments with interval [{}], current count is [{}]",
                 name,
-                interval,
-                n)
+                interval.toString,
+                n.toString)
```

Comment:
Hmm, ok, so has to be explicit. Kind of sucks as well because all toStrings will happen even if debug is not enabled, back to `.isDebugEnabled` I guess.

Reply:
yeah, for the varargs (>3 params) they must be of the same type in Scala.
Also something to ask Scala team about.

I agree that this is not good. I'll see if there is a better workaround, like putting `: Any` somewhere (probably works if placed on all of them).

Reply:
fixed (or at least improved) in https://github.com/akka/akka/pull/27605